templates/email/bexley/_problem-confirm_extra.html

```diff
@@ -0,0 +1,4 @@
+</p>
+<p style="[% p_style %]">
+  The report's reference number is <strong>[% report.id %]</strong>.
```

Comment:
I have a slight concern about including a reference number here, as the report isn't actually confirmed and hasn't been sent at this point. Giving a reference number to the user might lead them to believe they don't need to take any further action, but in fact the one thing they really need to do with this email is click the button.

Reply:
Do you think creating a fragment lower down - that appears after the link - which says something along the lines of "If you confirm your report, the report id will be xxxx if you wish to contact the council directly about it at a later date"

Reply:
I don't think we should display an ID at all for unconfirmed reports, to be honest. If we want to ensure the user always gets an ID when the report is sent you can use the `sub report_sent_confirmation_email { 'id' }` cobrand method, like we do for e.g. BathNES, which sends an email after the report's been sent.